client.go

```diff
@@ -72,13 +81,15 @@ type client struct {
 	seedBrokers []*Broker
 	deadSeeds   []*Broker
 
-	brokers  map[int32]*Broker                       // maps broker ids to brokers
-	metadata map[string]map[int32]*PartitionMetadata // maps topics to partition ids to metadata
+	brokers      map[int32]*Broker                       // maps broker ids to brokers
+	metadata     map[string]map[int32]*PartitionMetadata // maps topics to partition ids to metadata
+	coordinators map[string]*Broker                      // Maps consumer group names to coordinating brokers
```

Comment:
In fact there is a subtle bug in already: if a `RefreshMetadata()` call replaces a broker then the coordinator map will still contain a reference to the old one.